Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -176,34 +206,80 @@ struct SymbolGraphLoader {
+        let missingFallbackPlatforms = DefaultAvailability.fallbackPlatforms.filter {
+            !registeredPlatforms.contains($0.key) &&
+            !unconditionallyUnavailablePlatformNames.contains($0.key) &&
+            registeredPlatforms.contains($0.value)
+        }
+        // Platforms that are defined in the Info.plist that had no corresponding SGF
+        // and are not being added as fallback of another platform.
+        let missingAvailabilities = defaultAvailabilities.filter {
+                !missingFallbackPlatforms.keys.contains($0.platformName) &&
+                !registeredPlatforms.contains($0.platformName)
+        }
+        let symbolsWithFallbackAvailability = unifiedGraph.symbols.compactMapValues { symbol  -> UnifiedSymbolGraph.Symbol in
+            for (selector, _) in symbol.mixins {
+                if var symbolAvailability = (symbol.mixins[selector]?["availability"] as? SymbolGraph.Symbol.Availability) {
+                    guard !symbolAvailability.availability.isEmpty else { continue }
+                    // Add fallback availability.
+                    for (fallbackPlatform, inheritedPlatform) in missingFallbackPlatforms {
+                        if !symbolAvailability.availability.contains(where: { $0.domain?.rawValue == fallbackPlatform.rawValue }) {
+                            symbolAvailability.availability.forEach {
+                                // Add the platform fallback to the availability mixin the platform is inheriting from.
+                                // The added availability only copies the introduced version.
+                                if $0.domain?.rawValue.lowercased() == inheritedPlatform.rawValue.lowercased() {
+                                    var fallbackAvailability = $0
+                                    fallbackAvailability.deprecatedVersion = nil
+                                    fallbackAvailability.obsoletedVersion = nil
+                                    fallbackAvailability.domain = SymbolGraph.Symbol.Availability.Domain(rawValue: fallbackPlatform.rawValue)
+                                    symbolAvailability.availability.append(fallbackAvailability)
+                                }
+                            }
+                        }
+                    }
+                    // Add the missing default platform availability.
+                    missingAvailabilities.forEach { missingAvailability in
+                        if !symbolAvailability.availability.contains(where: { $0.domain?.rawValue == missingAvailability.platformName.rawValue }) {
+                            guard let defaultAvailability = AvailabilityItem(missingAvailability) else { return }
+                            symbolAvailability.availability.append(defaultAvailability)
+                        }
+                    }
+                    symbol.mixins[selector]!["availability"] = symbolAvailability
```

Comment:
This key is defined on each mixin kind
```suggestion
                    symbol.mixins[selector]![SymbolGraph.Symbol.Availability.mixinKey] = symbolAvailability
```

Reply:
Fixed here https://github.com/apple/swift-docc/commit/c55abc329895424275b96a3c47afef3f8c5bb3f9